DESCRIPTION

```diff
@@ -19,6 +19,7 @@ BugReports: https://github.com/Public-Health-Scotland/phstemplates/issues
 Depends: 
     R (>= 4.0.0)
 Imports: 
+    git2r,
```

Comment:
It looks like git2r has a lot of background changes - https://docs.ropensci.org/git2r/news/index.html, so it might be sensible to require at least the current version, to hopefully avoid any weird bugs...
```suggestion
    git2r (>= 0.36.2),
```